v7/latex/README.md

```diff
@@ -0,0 +1,42 @@
+This plugin allows to write posts and pages in a LaTeX-like syntax.
```

Comment:
I really dislike the *-like* part, especially because you did not specify what makes the syntax different from standard LaTeX.

(Pandoc can compile LaTeX to HTML, but the output is pretty ugly there, especially wrt math and spacing)